bin/node/src/commands.rs

```diff
@@ -0,0 +1,118 @@
+use crate::chain_spec::{
+    self, get_account_id_from_seed, AuthorityKeys, ChainParams, LOCAL_AUTHORITIES,
+};
+use aleph_primitives::AuthorityId as AlephId;
+use log::info;
+use sc_cli::{Error, KeystoreParams};
+use sc_keystore::LocalKeystore;
+use sc_service::config::{BasePath, KeystoreConfig};
+use sp_application_crypto::key_types;
+use sp_consensus_aura::sr25519::AuthorityId as AuraId;
+use sp_core::sr25519;
+use sp_keystore::SyncCryptoStore;
+use std::io::Write;
+use std::sync::Arc;
+use structopt::StructOpt;
+
+fn get_or_set_aura_key(keystore: &LocalKeystore) -> AuraId {
+    SyncCryptoStore::sr25519_public_keys(&*keystore, key_types::AURA)
+        .pop()
+        .map_or_else(
+            || SyncCryptoStore::sr25519_generate_new(&*keystore, key_types::AURA, None),
+            Ok,
+        )
+        .expect("Could not retrieve nor create Aura key")
+        .into()
+}
+
+fn get_or_set_aleph_key(keystore: &LocalKeystore) -> AlephId {
+    SyncCryptoStore::ed25519_public_keys(&*keystore, aleph_primitives::KEY_TYPE)
+        .pop()
+        .map_or_else(
+            || SyncCryptoStore::ed25519_generate_new(&*keystore, aleph_primitives::KEY_TYPE, None),
+            Ok,
+        )
+        .expect("Could not retrieve nor create Aleph key")
+        .into()
+}
+
+/// The `bootstrap-chain` command is used to generate private keys for the genesis authorities
+/// keys are written to the keystore of the authorities
+/// and the chain specification is printed to stdout in the JSON format
+#[derive(Debug, StructOpt)]
+pub struct BootstrapChainCmd {
+    /// Force raw genesis storage output.
+    #[structopt(long = "raw")]
+    pub raw: bool,
+
+    #[structopt(flatten)]
+    pub keystore_params: KeystoreParams,
+
+    #[structopt(flatten)]
+    pub chain_params: ChainParams,
+}
+
+impl BootstrapChainCmd {
+    pub fn run(&self) -> Result<(), Error> {
+        let chain_id = self.chain_params.chain_id();
+
+        let genesis_authorities = LOCAL_AUTHORITIES
+            .iter()
+            .map(|authority| {
+                let authority_keystore = self
+                    .open_keystore(authority, &chain_id)
+                    .unwrap_or_else(|_| panic!("Cannot open keystore for {}", authority));
+
+                let aura_key = get_or_set_aura_key(&authority_keystore);
+                let aleph_key = get_or_set_aleph_key(&authority_keystore);
+                let account_id = get_account_id_from_seed::<sr25519::Public>(authority);
```

Comment:
Don't know whether this should be fixed in this PR, but the account_id's should definitely not be generated from seed in case of testnet and mainnet (for dev runs we don't care much, seed is fine then). 
If you decide to not touch it here, then a high-priority jira task should be created for this -- to be done before the binary is finalized, I guess.

Reply:
https://cardinal-cryptography.atlassian.net/browse/A0-370